packages/@aws-cdk/aws-route53-patterns/lib/website-redirect.ts

```diff
@@ -107,4 +102,58 @@ export class HttpsRedirect extends Construct {
       new AaaaRecord(this, `RedirectAliasRecordSix${hash}`, aliasProps);
     });
   }
+
+  /**
+   * Gets the stack to use for creating the Certificate
+   * If the current stack is not in `us-east-1` then this
+   * will create a new `us-east-1` stack.
+   */
+  private get certificateScope(): Construct {
+    const stack = Stack.of(this);
+    const parent = stack.node.scope;
+    if (!parent) {
+      throw new Error(`Stack ${stack.stackId} must be created in the scope of an App or Stage`);
+    }
+    if (Token.isUnresolved(stack.region)) {
+      throw new Error(`When ${ROUTE53_PATTERNS_USE_CERTIFICATE} is enabled, a region must be defined on the Stack`);
+    }
+    if (stack.region !== 'us-east-1') {
+      const stackId = `certificate-redirect-stack-${stack.node.addr}`;
+      const certStack = parent.node.tryFindChild(stackId) as Stack;
+      return certStack ?? new Stack(parent, stackId, {
+        env: { region: 'us-east-1', account: stack.account },
+      });
+    }
+    return this;
+  }
+
+  /**
+   * Creates a certificate.
+   *
+   * If the `ROUTE53_PATTERNS_USE_CERTIFICATE` feature flag is set then
+   * this will use the `Certificate` class otherwise it will use the
+   * `DnsValidatedCertificate` class
+   *
+   * This is also safe to upgrade since the new certificate will be created and updated
+   * on the CloudFront distribution before the old one is deleted.
```

Comment:
this information should be replicated to the readme.

Reply:
Good call.